src/components/PageBanner/index.js

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ResponsiveImage from 'components/ResponsiveImage';
+import { FULL_WIDTH_RESPONSIVE_IMAGE_SIZES } from 'js/helpers/constants';
 
-const PageBanner = ({ imageUrl, imageTitle }) => (
-  <div
+const PageBanner = ({ imagesPath, imageFilename, imageExtension, imageTitle }) => (
```

Comment:
I think this is going to break storybook -- could you update the PageBanner.stories.js too